dll/cpl/timedate/internettime.c

```diff
@@ -319,6 +219,41 @@ OnInitDialog(HWND hwnd)
     CreateNTPServerList(hwnd);
 }
 
+static VOID
+OnAutoSync(BOOL Sync)
+{
+    HKEY hKey;
+    LONG lRet;
+    LPCWSTR szAuto;
+
+    if (Sync)
+        szAuto = L"NTP";
+    else
+        szAuto = L"NoSync";
+
+    lRet = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
+                         L"SYSTEM\\CurrentControlSet\\Services\\W32Time\\Parameters",
+                         0,
+                         KEY_SET_VALUE,
+                         &hKey);
+        if (lRet != ERROR_SUCCESS)
```

Comment:
Indentation seems to be messed up here. Will fix.

Reply:
Thanks.